concent_api/core/management/commands/concent_adapt_data_to_current_version.py

```diff
@@ -0,0 +1,29 @@
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+from typing import Any
+
+from common.helpers import deserialize_message
+from core.models import StoredMessage
+from core.models import Subtask
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Adapts whole database to current Concent's version"
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        for subtask in Subtask.objects.all():
+            try:
+                report_computed_task =  deserialize_message(subtask.report_computed_task.data.tobytes())
+                if not isinstance(report_computed_task.size, int):
+                    delete_unsupported_messages(subtask.subtask_id)
+            except:
+                delete_unsupported_messages(subtask.subtask_id)
+
+
+def delete_unsupported_messages(subtask_id: str) -> None:
+    Subtask.objects.filter(subtask_id=subtask_id).delete()
```

Comment:
Instead of those two lines you can do (if you pass `subtask` instead of `subtask_id`):
```
StoredMessage.objects.filter(subtask_id=subtask.subtask_id).delete()
subtask.delete()
```

That would be one query less.

Reply:
I didn't want to pass whole subtask to this function that's why I passed there only `subtask_id`. Do you want me to change it?

Reply:
Yes please, it will result in a one query less with each call of this function (you will not retrieve `Subtask` again from database but use the one which you already have) which is some optimization and good practice.